client/pages/state-form.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+export default class StateForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: ''
+    };
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event) {
+    this.setState({
+      value: event.target.value
+    });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    location.href = '#parks-by-state';
```

Comment:
We only want to target the `location.hash` in this scenario. `location.href` works, but it's not the intent.